src/task/JoinAndReceiveIndexTask.ts

```diff
@@ -48,19 +48,20 @@ export default class JoinAndReceiveIndexTask extends BaseTask {
 
         handleSignalingClientEvent(event: SignalingClientEvent): void {
           if (event.type === SignalingClientEventType.WebSocketClosed) {
-            context.logger.warn(
-              `signaling connection closed by server with code ${event.closeCode} and reason: ${event.closeReason}`
-            );
+            let message = `The signaling connection was closed with code ${event.closeCode} and reason: ${event.closeReason}`;
+            context.logger.warn(message);
+
             let statusCode: MeetingSessionStatusCode = MeetingSessionStatusCode.SignalingBadRequest;
             if (event.closeCode === 4410) {
               context.logger.warn(`the meeting cannot be joined because it is has been ended`);
               statusCode = MeetingSessionStatusCode.MeetingEnded;
+              message = 'The meeting already ended.';
```

Comment:
Should this message be the same as the `context.logger.warn` which is "the meeting cannot be joined because it is has been ended"?

Reply:
I pushed a revision for replacing `the meeting cannot be joined...` with the same message `The meeting already ended.`. Since JS SDK offers `start` and `stop` APIs, I removed "joined" from the message.